### PR TITLE
yoav - cleaned and added temperature

### PR DESCRIPTION
**Pull request by YHerm**

### CODE REVIEW

File: src/main/java/edu/greenblitz/robotName/subsystems/Shooter/Pivot/FalconPivot/FalconPivotConstants.java
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/robotName/subsystems/Shooter/Pivot/FalconPivot/FalconPivotConstants.java:1]: why not make this a class in PivotConstants?

File: src/main/java/edu/greenblitz/robotName/subsystems/Shooter/Pivot/SimulationPivot/SimulationPivotConstants.java
Comment (greenblitz4590) [inline src/main/java/edu/greenblitz/robotName/subsystems/Shooter/Pivot/SimulationPivot/SimulationPivotConstants.java:1]: again why is this not in PivotConstants

File: src/main/java/edu/greenblitz/robotName/subsystems/Shooter/Pivot/FalconPivot/FalconPivotConstants.java
Comment (YHerm) [reply src/main/java/edu/greenblitz/robotName/subsystems/Shooter/Pivot/FalconPivot/FalconPivotConstants.java:1]: It helps people who don’t familiar with the code find things they want

File: src/main/java/edu/greenblitz/robotName/subsystems/Shooter/Pivot/SimulationPivot/SimulationPivotConstants.java
Comment (YHerm) [reply src/main/java/edu/greenblitz/robotName/subsystems/Shooter/Pivot/SimulationPivot/SimulationPivotConstants.java:1]: Noam said

Comment (greenblitz4590) [reply src/main/java/edu/greenblitz/robotName/subsystems/Shooter/Pivot/SimulationPivot/SimulationPivotConstants.java:1]: ok

File: src/main/java/edu/greenblitz/robotName/subsystems/Shooter/Pivot/FalconPivot/FalconPivotConstants.java
Comment (greenblitz4590) [reply src/main/java/edu/greenblitz/robotName/subsystems/Shooter/Pivot/FalconPivot/FalconPivotConstants.java:1]: i dont think it really helps but ok

**Pull request merged**